src/servers/api-server.ts

```diff
@@ -41,6 +49,14 @@ apiServer.get("/api/resources/datasources", async (c) => {
     return c.json(datasources);
 });
 
+apiServer.get("/api/resources/datasource-id", async (c) => {
```

Comment:
this isn't exposed as a resource in the mcp, so shouldnt be exposed here on a `resources/` path for consistency